first_5_steps/1_install_tt_buda.md

```diff
@@ -63,16 +63,37 @@ The [tt-firmware](https://github.com/tenstorrent/tt-firmware) file needs to be i
 
 ### Backend Compiler Dependencies
 
-Instructions to install the Tenstorrent backend compiler dependencies on a fresh install of Ubuntu Server 20.04.
+Instructions to install the Tenstorrent backend compiler dependencies on a fresh install of Ubuntu Server 20.04 or Ubuntu Server 22.04.
 
 *You may need to **append** each `apt-get` command with `sudo` if you do not have root permissions.*
 
+For both operating systems run the following commands:
+
+```bash
+apt update -y
+apt upgrade -y --no-install-recommends
+apt install -y build-essential curl libboost-all-dev libgl1-mesa-glx libgoogle-glog-dev libhdf5-serial-dev ruby software-properties-common libzmq3-dev 
+
+```
+
+For Ubuntu 20.04, add:
+```bash
+apt install -y clang-6.0 libyaml-cpp-dev libzmq3-dev python3.8-venv 
+```
+
+For Ubuntu 22.04, add:
+```bash
+apt install -y python3-venv clang wget
```

Comment:
Can you test if `python3-venv` will work on Ubuntu 20.04? This way we can make it a common package for both.

Reply:
done https://github.com/tenstorrent/tt-buda-demos/pull/38#:~:text=Verified-,368eae3,-Merge%20state
consolidates all